movfs4l.py

```diff
@@ -767,6 +807,7 @@ def generate_config(variables, inipath, config=None):
 
     with open(inipath, 'w') as inifile:
         config.write(inifile)
+        inifile.close()
```

Comment:
The idea with the `with ...` is for close to be unnecessary: https://www.geeksforgeeks.org/with-statement-in-python/, even if an exception is present :)

Reply:
Good to know, thanks!